### ISSUE TITLE
Site is not working. Uncaught error in private Firefox window and wss connection establish fail in normal window.

### ISSUE THREAD

**Issue by slrslr** [ipfs-shipyard/peerpad#334]
https://ipfs.io/ipns/peerpad.net/#/ redirects to [here](http://peerpad.net.ipns.localhost:8080/#/w/markdown/9J85gBriZ2dptgmUL8SaS3YhdGcBZgt1jvGHVFjjLK3D/4XTTM37X5zJZzgrjpz3UskjuXk2CyY8GJKjjdX8megQCAkKvt-K3TgUHZu4xTmLsGD6jpUZr21uhmigg8WG17wsjGaUcYgSDMupvC4cZQDQu1AQZemkecMj8dfNVwTi1svq9ZixcNQqJzYuEhNfVh1YbtcR3ymxFcffTqPdeXabUjBEYVpNLtAHBoa) in Firefox 93.0 web browser private window. Dev. console shows Uncaught error:
![peerpaduncaught](https://user-images.githubusercontent.com/6596726/138870242-88df6667-a358-4952-9155-3e85c7fd83f9.png)
Yet in normal Firefox window is not this error instead "only" wss errors:
![peerpadnormalwindowwssestablishissue](https://user-images.githubusercontent.com/6596726/138870571-c2f69716-0f1f-4444-a722-bdf3b3b71b9e.png)
I am using IPFS companion 0.9.1/dc2715a

**Comment by jbenet** [comment]
Yeah, i'm seeing the same -- site broke -- maybe libp2p infra is missing / changed?
